### PR TITLE
FOGL-5018: Removed foglamp related stuff from version history file

### PR DESCRIPTION
**Pull request by YashTatkondawar**

### CODE REVIEW

**Review by ashish-jabble — CHANGES_REQUESTED** [review]
There are some typo errors: retirgger, perviously

And I see there are some bug fixes which falls under New features section

**Fledge Core**
a)  The REST API for the notification service was missing the re-trigger time information for configured notification in the retrieval and update calls. This has now been added.
b) The notification service previously logged errors if not delivery no notification plugins had been installed. This is no longer the case.

Also not sure _documentation point_ one's are really a new feature?

if yes then  Move point "Documentation has been added for configuration of the storage service." which is under BUG section at the moment.

**Plugins**
a) A memory issue with the python 35 filter integration has been resolved.
b) Packaging conflicts between plugins that used the same additional libraries have been resolved allow both plugins to be install on the same machine. This issue impacted the plugins that used MQTT as a transport layer.
c) The OPC/UA north plugin did not correctly handle the types for integer data, this has now been resolved.
d) The OPC/UA south plugin did not allow subscriptions to integer node ids. This has now been added.
e) A problem with reading multiple modbus input registers into a single value has been resolved in the ModbusC plugin

Under New features or new section Improvements?
a) The modbus plugin has been enhanced to support Modbus over IPv6, also request timeout has been added as a configuration option. There have been improvements to the error handling also.

cc/ @MarkRiddoch

**Review by MarkRiddoch — APPROVED** [review]

**Comment by YashTatkondawar** [comment]
> Also not sure _documentation point_ one's are really a new feature?
> if yes then Move point "Documentation has been added for configuration of the storage service." which is under BUG section at the moment.
Moved to new feature

> Under New features or new section Improvements?
> a) The modbus plugin has been enhanced to support Modbus over IPv6, also request timeout has been added as a configuration option. There have been improvements to the error handling also.
Kept this under Bug Fix as it was a fix.

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -78,31 +79,23 @@ Release Date: 2021-02-10
...
        - The modbus plugin has been enhanced to support Modbus over IPv6, also request timeout has been added as a configuration option. There have been improvements to the error handling also.
        - The DNP3 south plugin incorrectly treated all data as strings, this meant it was not easy to process it the data with generic plugins. This has now been resolved and data is treated as floating point or integer values.
-       - The OMF north plugin perviously reported the incorrect version information. This has now been resolved.
+       - The OMF north plugin previously reported the incorrect version information. This has now been resolved.
+       - A memory issue with the python 35 filter integration has been resolved.
+       - Packaging conflicts between plugins that used the same additional libraries have been resolved allow both plugins to be install on the same machine. This issue impacted the plugins that used MQTT as a transport layer.
+       - The OPC/UA north plugin did not correctly handle the types for integer data, this has now been resolved.
+       - The OPCUA south plugin did not allow subscriptions to integer node ids. This has now been added.
```
Comment (ashish-jabble) [inline docs/91_version_history.rst:97]: OPC/UA

**Review by ashish-jabble — APPROVED** [review]
Thanks. LGTM!

**Pull request merged**